extensions/amp-story/amp-story.md

```diff
@@ -156,6 +193,22 @@ The `amp-story` component represents an entire story.  The component itself  imp
 
 Identifies that the AMP document is a story.
 
+##### title [required]
```

Comment:
If these are required, it would make sense to update the sample in the example section (right before this section) so it shows those attributes.

Reply:
Done.